lib.Makefile

```diff
@@ -262,7 +262,11 @@ REPO_ROOT := $(shell git rev-parse --show-toplevel)
 CERTS_PATH := $(REPO_ROOT)/hack/test/certs
 
 # The image to use for building calico/base-dependent modules (e.g. apiserver, typha).
+ifdef USE_UBI_AS_CALICO_BASE
+CALICO_BASE ?= $(UBI_IMAGE)
```

Comment:
Will this be the `-minimal` version that you were using before?

Reply:
Yes we seem to use `ubi-minimal` in all these areas